src/components/RadioButtons/RadioButtons.js

```diff
@@ -69,10 +33,11 @@ function RadioButton({
           name={name}
           value={value}
           checked={checked}
+          onClick={onClick}
+          onChange={onChange}
           required={required}
           disabled={disabled}
           data-tid={rest['data-tid']}
-          {...rest} // for redux-form input.onChange, etc.
```

Comment:
🙌

Reply:
I think we can even take out the `{...rest}`, unless we want it for some reason. I think it was only for redux-form.